concepts/auth_overview.md

```diff
@@ -4,6 +4,8 @@ To call Microsoft Graph, your app must acquire an access token from Azure Active
 
 This topic provides an overview of access tokens, Azure AD, and how your app can get access tokens. If you are already familiar with integrating an app with Azure AD to get tokens, then you can skip ahead to [Next Steps](#next-steps) for information and samples specific to Microsoft Graph. 
 
+> **Important:**  How conditional access policies apply to Microsoft Graph is changing. Applications need to be updated to handle scenarios where conditional access policies are configured. For more information and guidance, see [FWLink that Jim was going to create]()
```

Comment:
The actual link should be here, right?

Reply:
@jesakowi Yes.  When we have the link, I'll insert it in the parenthesis.